test/test-core.js

```diff
@@ -711,6 +712,80 @@ describe('Test log-core', function () {
         });
     });
 
+    describe('Test getTenantId', function () {
+        var core = rewire("../cf-nodejs-logging-support-core/log-core.js");
+        var logObject = null;
+        var getTenantId = null;
+
+        before(function () {
+            core.__set__({
+                "sendLog": function (level, logObj) {
+                    logObject = logObj;
+                }
+            });
+            core.setConfig(importFresh("../config.js").config);
+
+            getTenantId = core.__get__("getTenantId");
+        });
+
+        it('Test correct reading from request', function () {
+            var testRequest = {};
+            testRequest.logObject = {
+                "tenant_id": "456"
```

Comment:
If we enforce the use of a valid uuid, this testing id should only work if it is a correct uuid.